pybamm/input/parameters/lithium_ion/lithium_platings/okane2022_Li_plating/parameters.csv

```diff
@@ -0,0 +1,14 @@
+Name [units],Value,Reference,Notes
+,,,
+,,,
+# Lithium plating parameters,,,
+Lithium metal partial molar volume [m3.mol-1],1.30E-05,Yang2017,
+Lithium plating kinetic rate constant [m.s-1],1E-9,,
+Exchange-current density for plating [A.m-2],[function]plating_exchange_current_density_OKane2020,,
+Exchange-current density for stripping [A.m-2],[function]stripping_exchange_current_density_OKane2020,,
+Initial plated lithium concentration [mol.m-3],0.00E+00,,
+Typical plated lithium concentration [mol.m-3],1000,1 molar,
+Lithium stripping transfer coefficient,0.35,,
+Lithium plating transfer coefficient,0.65,,
+Dead lithium decay constant [s-1],1E-6,,
+Dead lithium decay rate [s-1],[function]SEI_limited_dead_lithium_OKane2022,,
```

Comment:
So this parameter set is almost a superset of O'Kane 2020, except the rate constant changes from 1e-10 to 1e-9?

Reply:
Also, the transfer coefficients are different